README.md

```diff
@@ -22,7 +22,8 @@ Here are the requirements for Magepack to work:
 1. You need Node.js version 10 or higher installed.
 2. If you are using Magento 2.3.5 or lower, you need to have `mixins.js` module patched [(patch provided and explained here)](https://github.com/magento/baler/issues/23).
 3. If you are using Magento 2.3.3 or lower, you need `jquery.cookie` module shim [(patch provided and explained here)](https://github.com/magento/baler/issues/6).
-4. [Magepack Magento module](https://github.com/magesuite/magepack-magento) installed.
+4. If you are using Magento 2.3.6-p1 or lower, you need `JSFooterPlugin` patched [(patch provided and explained here)](https://github.com/magento/magento2/pull/27270).
```

Comment:
Hey, thanks for contribution!

As far as I know this patch is not required for shorter pages but it is useful information nonetheless. Maybe we could change the wording that _it is recommended_ to have this plugin patched?